steps/create_image.cwl

```diff
@@ -0,0 +1,151 @@
+class: CommandLineTool
+cwlVersion: v1.0
+$namespaces:
+  s: 'http://schema.org/'
+  sbg: 'https://www.sevenbridges.com'
+id: create_image
+
+hints:
+  DockerRequirement:
+      dockerImageId: kernsuite/prefactor
+
+baseCommand: [python]
+inputs:
+  - id: mapfile_deep_low_size
+    type: File
+  - id: mapfile_nchansout_clean1
+    type: File
+  - id: mapfile_nwavelengths_low
+    type: File
+  - id: image_padding
+    type: float
+  - id: maxlambda_lowres
+    type: int
+  - id: cellsize_lowres_deg
+    type: float
+  - id: numcpu
```

Comment:
Is this number of cores? We should probably put a default value using `RuntimeRequirements`; users can override using https://github.com/common-workflow-language/cwltool/#overriding-workflow-requirements-at-load-time (which will be part of CWL v1.1 proper)

Reply:
Yeah, I guessed you'd be suggesting be something like that. Would we be able to use something comparable for the memory percentage to use as well? Will this actually limit the number of cores used at the docker run level actually?

Reply:
So what I want is: 
- all cores unless set otherwise
- 95% of RAM unless set otherwise (ok, you haven't commented on that setting but I think it would be neat to drive that via a ResourceRequirement as well actually)

The first may actually be default behaviour if nothing is set.

Reply:
for my reference: will switch wclean from using -mem to -abs-mem, supporting an absolute memory constraint.